homepage1.html

```diff
@@ -1,241 +0,0 @@
-<!DOCTYPE HTML>
-<html>
-
-<head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-    <link rel="stylesheet" href="homepage1.css" />
-</head>
```

Comment:
This file should be named `index.html` It shouldn't matter what you name your files, but for your homepage it really does